### ISSUE TITLE
The model cannot converge when `--update_freq` is used in fairseq

### ISSUE THREAD

**Issue by KelleyYin** [bytedance/lightseq#138]
In the process of reproducing wmt14_en_de experiment, I found that `loss=inf` appeared after adding `--update_freq=16`. On the contrary, the `loss` becomes normal after removing the `--update_freq`.

Have you ever experimented with the parameter `--update_freq` in  the training process?

**Comment by godweiyang** [comment]
Sorry that I can not reproduce your bug. I ran on 8 V100 GPUs and also used `--update-freq 16`. However, I obtained the same results between LightSeq and Fairseq, both without `loss=inf`.
So can you tell me the environment of you experiments (number of GPUs, CUDA and torch version, etc.)?

**Comment by godweiyang** [comment]
> In the process of reproducing wmt14_en_de experiment, I found that `loss=inf` appeared after adding `--update_freq=16`. On the contrary, the `loss` becomes normal after removing the `--update_freq`.
> 
> Have you ever experimented with the parameter `--update_freq` in the training process?

Sorry that I can not reproduce your bug. I ran on 8 V100 GPUs and also used --update-freq 16. However, I obtained the same results between LightSeq and Fairseq, both without loss=inf.
So can you tell me the environment of you experiments (number of GPUs, CUDA and torch version, etc.)?

**Comment by KelleyYin** [comment]
Thanks for your reply !

Environments as follows:
* pytorch: 1.7.1+cu110
* fairseq: v0.10.2
* CUDA: 11.0
* GPUs: 8 * V100
* gcc: 7.3.1

**Comment by godweiyang** [comment]
> Thanks for your reply !
> 
> Environments as follows:
> 
> * pytorch: 1.7.1+cu110
> * fairseq: v0.10.2
> * CUDA: 11.0
> * GPUs: 8 * V100
> * gcc: 7.3.1

Your environment is the same as mine.

You can try the following commands as me:
```shell
git clone https://github.com/bytedance/lightseq.git
cd lightseq
pip3 install -e .
```

And then add `--update-freq 16` in the last line of `examples/training/fairseq/ls_fairseq_wmt14en2de.sh`. Then run `sh examples/training/fairseq/ls_fairseq_wmt14en2de.sh`.

I ran twice and both obtain normal results:
